xbmc/video/VideoDatabase.cpp

```diff
@@ -6249,8 +6287,9 @@ bool CVideoDatabase::GetYearsNav(const std::string& strBaseDir, CFileItemList& i
       }
       else if (idContent == VIDEODB_CONTENT_MUSICVIDEOS)
       {
-        strSQL = "select musicvideo_view.premiered, path.strPath, files.playCount from musicvideo_view ";
+        strSQL = "select musicvideo_view.premiered, path.strPath, COUNT(history.idFile) from musicvideo_view ";
         extFilter.AppendJoin("join files on files.idFile = musicvideo_view.idFile join path on files.idPath = path.idPath");
+        extFilter.AppendJoin("JOIN history ON history.idFile = movie_view.idFile");
```

Comment:
Check if we can use subqueries here instead, as inner join might hide some values